Create rule S7435: Processing persistent unique identifiers is security-sensitive (#4776)
SonarSource/rspec: rules/S7435/description.adoc

> Mobile devices expose unique identifiers that can be used to identify users accros applications or devices. These identifiers put user privacy at risk when they allow tracking user activity without their consent or makes it difficult or impossible for users to reset them.

Reviewer: ```suggestion
Mobile devices expose unique identifiers that can be used to identify users across applications or devices. These identifiers put user privacy at risk, as they might allow the tracking of user activity without consent, while making it difficult or impossible for users to reset them.
```

> Privacy violations can cause app to be removed from app stores, legal actions, or loss of trust from users.

Reviewer: ```suggestion
Privacy violations can cause apps to be removed from app stores and can result in legal action or loss of trust from users.
```
